## 75% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 11
Expected to fail tests: 2
Passed tests: 38
```
```
IDs of failed tests: 727230,702947,702786,702731,703202,702841,702809,704615,702782,727229,703503 
```
```
IDs of expected to fail tests: 727232,703495 
```
<h3>Failed tests (11)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/community-channel-not-opening-first-time&PR_ID=20508&APK_NAME=20508.apk&TR_CASE_IDS=727230,702947,702786,702731,703202,702841,702809,704615,702782,727229,703503">Rerun failed tests</a></li><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589857">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'This is a test message to check some reactions.')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'This is a test message to check some reactions.')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div></blockquote></p>

```
critical/chats/test_group_chat.py:125: in test_group_chat_reactions
    self.loop.run_until_complete(run_in_parallel((
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
critical/chats/test_group_chat.py:121: in _check_reactions_count
    chat_element.emojis_below_message(emoji="thumbs-up").wait_for_element_text(2)
../views/base_element.py:190: in wait_for_element_text
    self.driver.fail(message if message else "`%s` is not equal to expected `%s` in %s sec" % (
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 2: `1` is not equal to expected `2` in 30 sec
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/920d078c902f4b93b26a9323441e7990?auth=6e0cce8946f2374b69b723a353c44d3d#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0b070bfe2f845518fcd92b6d4e0e6ba?auth=41e575c54ecd645eedb5a46dafa57814#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f8dc77a13b634eeab3f99b4b65139ffe?auth=333669d4e6131a92bd43aa812fea4bd1#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589863">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div># STEP: Check Status community tags in the Discover communities screen</div><div># STEP: Check Status Inu community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:300: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community ' 0xUX' is not in the Discover Communities list.
E    Community 'Status' is not in the Discover Communities list.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d1cd4303a745dc9fc200f866258aee?auth=e7de1a4ab730c2a79c0e9aea2dc3591a#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589821">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:198: in test_1_1_chat_pin_messages
    chat.pinned_messages_count.wait_for_element_text(text="2",
../views/base_element.py:190: in wait_for_element_text
    self.driver.fail(message if message else "`%s` is not equal to expected `%s` in %s sec" % (
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Pinned messages count is not 2 as expected!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#193">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589828">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><p><blockquote><div>Device 1: Long press on `ChatElementByText` until expected element is shown </div><div>Device 1: Find `Button` by `xpath`: `//*[@text="Reply"]` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:129: in test_1_1_chat_emoji_send_reply_and_open_link
    self.chat_1.quote_message(emoji_unicode)
../views/chat_view.py:1041: in quote_message
    self.reply_message_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@text="Reply"]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#155">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589839">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><p><blockquote><div># STEP: Check new messages badge is shown for channel</div><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:667: in test_community_unread_messages_badge
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 New message community badge is not shown
E    New messages channel badge is not shown on channel
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#400">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#363">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589866">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><p><blockquote><div># STEP: Receiver is checking if initial messages were delivered</div><div>Device 1: Looking for a message by text: message to edit </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:811: in test_community_edit_delete_message_when_offline
    self.channel_1.driver.fail("Message '%s' was not received" % message)
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Message 'message to edit' was not received
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#595">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#495">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589868">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:152: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:40: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589867">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:132: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['wallet_address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:130: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:118: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:112: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589850">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `LogInButton` by `accessibility id`: `login-button` </div><div>Device 2: Tap on found: LogInButton </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:274: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9838080a3a414007ac44b1ef45861814?auth=677fb117bfeeb6e7dd2207c522cfa45a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dccd5dcd19894e4fabd1f179d9977358?auth=e8e1e5b1b617dfe156466c71add931fa">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589831">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div># STEP: Invited member gets push notification with the mention and tap it</div><div>Device 2: Getting PN by 'user_2' </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:957: in test_community_mentions_push_notification
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Push notification with the mention was not received by admin
E    Edited message is not shown correctly for the (receiver) admin
E    Push notification with the mention was not received by the invited member
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c40c5e85f4304f5480f19ee01a7172f7?auth=5e3bc3331b91112793bd402e583360e3#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/26f828414dc64069b335c89071a978e9?auth=1613e096ad88443c9c538ccfdec8065d#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589834">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: quote reply (one row) </div><div>Device 2: Looking for a message by text: quote reply (one row) </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1009: in test_community_markdown_support
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 italic text in underscore is not displayed with markdown in community channel for the recipient (device 1)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c40c5e85f4304f5480f19ee01a7172f7?auth=5e3bc3331b91112793bd402e583360e3#196">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/26f828414dc64069b335c89071a978e9?auth=1613e096ad88443c9c538ccfdec8065d#170">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (2)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589861">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:497: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/920d078c902f4b93b26a9323441e7990?auth=6e0cce8946f2374b69b723a353c44d3d#391">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0b070bfe2f845518fcd92b6d4e0e6ba?auth=41e575c54ecd645eedb5a46dafa57814#252">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f8dc77a13b634eeab3f99b4b65139ffe?auth=333669d4e6131a92bd43aa812fea4bd1#217">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589870">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:256: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6a47a2c6a6b845679a5e0a29d5941bac?auth=67238e6561bb615e1ef1e46200169f28#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (38)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589862">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/096127f776db406ebb72ff95fa539026?auth=3776c09ff1708c6ff82babbf81653ee6#110">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e0c86098c3e14020b0da64d9cac3dcc3?auth=0bde65a361a28e696071bf4a48a3b2d8#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589829">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/096127f776db406ebb72ff95fa539026?auth=3776c09ff1708c6ff82babbf81653ee6#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e0c86098c3e14020b0da64d9cac3dcc3?auth=0bde65a361a28e696071bf4a48a3b2d8#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589830">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/096127f776db406ebb72ff95fa539026?auth=3776c09ff1708c6ff82babbf81653ee6#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e0c86098c3e14020b0da64d9cac3dcc3?auth=0bde65a361a28e696071bf4a48a3b2d8#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589832">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/920d078c902f4b93b26a9323441e7990?auth=6e0cce8946f2374b69b723a353c44d3d#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0b070bfe2f845518fcd92b6d4e0e6ba?auth=41e575c54ecd645eedb5a46dafa57814#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f8dc77a13b634eeab3f99b4b65139ffe?auth=333669d4e6131a92bd43aa812fea4bd1#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589833">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/920d078c902f4b93b26a9323441e7990?auth=6e0cce8946f2374b69b723a353c44d3d#237">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0b070bfe2f845518fcd92b6d4e0e6ba?auth=41e575c54ecd645eedb5a46dafa57814#178">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f8dc77a13b634eeab3f99b4b65139ffe?auth=333669d4e6131a92bd43aa812fea4bd1#176">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589822">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/920d078c902f4b93b26a9323441e7990?auth=6e0cce8946f2374b69b723a353c44d3d#271">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0b070bfe2f845518fcd92b6d4e0e6ba?auth=41e575c54ecd645eedb5a46dafa57814#205">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f8dc77a13b634eeab3f99b4b65139ffe?auth=333669d4e6131a92bd43aa812fea4bd1#203">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589858">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/920d078c902f4b93b26a9323441e7990?auth=6e0cce8946f2374b69b723a353c44d3d#200">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0b070bfe2f845518fcd92b6d4e0e6ba?auth=41e575c54ecd645eedb5a46dafa57814#142">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f8dc77a13b634eeab3f99b4b65139ffe?auth=333669d4e6131a92bd43aa812fea4bd1#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589853">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/10cb3f1ded4248ee9876d20603ac0148?auth=a02b9990537199bdec7e5ba6a237c2ea#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/944ae895d4da4631aceeeca0698f5793?auth=7fcdb4bfc86bbca587049fff6b9fd590#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589852">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/10cb3f1ded4248ee9876d20603ac0148?auth=a02b9990537199bdec7e5ba6a237c2ea#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/944ae895d4da4631aceeeca0698f5793?auth=7fcdb4bfc86bbca587049fff6b9fd590#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589845">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8aabad8d3a834d2185b2fb62a5a39c77?auth=ddc2b153032fcb748938a65a6dad25c7#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/23cc9102d31549bdb917b57ace5ed220?auth=0731c178820379f399655e23d5c6a4c1#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589844">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8aabad8d3a834d2185b2fb62a5a39c77?auth=ddc2b153032fcb748938a65a6dad25c7#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/23cc9102d31549bdb917b57ace5ed220?auth=0731c178820379f399655e23d5c6a4c1#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589827">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8aabad8d3a834d2185b2fb62a5a39c77?auth=ddc2b153032fcb748938a65a6dad25c7#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/23cc9102d31549bdb917b57ace5ed220?auth=0731c178820379f399655e23d5c6a4c1#218">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589824">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d1cd4303a745dc9fc200f866258aee?auth=e7de1a4ab730c2a79c0e9aea2dc3591a#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589843">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d1cd4303a745dc9fc200f866258aee?auth=e7de1a4ab730c2a79c0e9aea2dc3591a#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589855">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d1cd4303a745dc9fc200f866258aee?auth=e7de1a4ab730c2a79c0e9aea2dc3591a#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589848">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d1cd4303a745dc9fc200f866258aee?auth=e7de1a4ab730c2a79c0e9aea2dc3591a#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589859">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d1cd4303a745dc9fc200f866258aee?auth=e7de1a4ab730c2a79c0e9aea2dc3591a#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589838">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#311">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#285">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589849">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#433">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#393">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589854">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#560">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#469">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589841">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#335">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#302">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589837">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#200">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#135">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589836">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589847">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#262">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#225">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589840">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589856">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aa49d3f0b6b4193ba25be7a757be775?auth=4796c0c89902f186533181f861e1bdfb#237">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92d0893f05e943e8bfa2f80bf570c798?auth=a49ff0774aa8aac03b30c2fa34dbbf84#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589869">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6a47a2c6a6b845679a5e0a29d5941bac?auth=67238e6561bb615e1ef1e46200169f28#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589826">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ca713d9879754a528ff8c53c79a0459f?auth=3748054db68f7a95f64b0405733a62ed#203">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589865">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ca713d9879754a528ff8c53c79a0459f?auth=3748054db68f7a95f64b0405733a62ed#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589846">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#345">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#232">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589820">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589825">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#238">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#151">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589823">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#437">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#342">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589835">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#296">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589860">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c0395178e3c44e8a893cd32a623a0464?auth=524cb3af875402a158124f59bba0376f#376">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/20f3111a8e3c497e8ba1b2ac60659cbb?auth=fe557e1c600721b7ccaa28a19e448c2e#282">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589842">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c40c5e85f4304f5480f19ee01a7172f7?auth=5e3bc3331b91112793bd402e583360e3#314">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/26f828414dc64069b335c89071a978e9?auth=1613e096ad88443c9c538ccfdec8065d#277">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589851">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c40c5e85f4304f5480f19ee01a7172f7?auth=5e3bc3331b91112793bd402e583360e3#317">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/26f828414dc64069b335c89071a978e9?auth=1613e096ad88443c9c538ccfdec8065d#299">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589864">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c40c5e85f4304f5480f19ee01a7172f7?auth=5e3bc3331b91112793bd402e583360e3#514">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/26f828414dc64069b335c89071a978e9?auth=1613e096ad88443c9c538ccfdec8065d#476">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/communities/events.cljs
+++ b/src/status_im/contexts/communities/events.cljs
@@ -230,7 +230,7 @@
   (when community
     {:db (update db :communities/fetching-communities dissoc community-id)
      :fx [[:dispatch [:communities/handle-community community]]
-          [:dispatch [:communities/update-last-opened-at community-id]]
+          [:dispatch [:chat.ui/spectate-community community-id]]
           [:dispatch
            [:chat.ui/cache-link-preview-data (link-preview.events/community-link community-id)
             community]]]}))
@@ -251,7 +251,7 @@
               :error err}))
 
 (defn fetch-community
-  [{:keys [db]} [{:keys [community-id update-last-opened-at?]}]]
+  [{:keys [db]} [{:keys [community-id]}]]
   (when (and community-id
              (not (get-in db [:communities community-id]))
              (not (get-in db [:communities/fetching-communities community-id])))
@@ -262,11 +262,8 @@
                                     :WaitForResponse true}]
                       :on-success (fn [community]
                                     (if community
-                                      (do (when update-last-opened-at?
-                                            (rf/dispatch [:communities/update-last-opened-at
-                                                          community-id]))
-                                          (rf/dispatch [:chat.ui/community-fetched community-id
-                                                        community]))
+                                      (rf/dispatch [:chat.ui/community-fetched community-id
+                                                    community])
                                       (failed-to-fetch-community
                                        community-id
                                        "community wasn't found at the store node")))
@@ -280,8 +277,7 @@
      [:schema
       [:vector
        [:map {:closed true}
-        [:community-id {:optional true} :string]
-        [:update-last-opened-at? {:optional true} [:maybe :boolean]]]]]]]
+        [:community-id {:optional true} :string]]]]]]
    [:maybe
     [:map
      [:db map?]
@@ -340,10 +336,9 @@
       (rf/merge
        cofx
        {:fx [[:dispatch [:chat.ui/spectate-community community-id]]
+             [:dispatch [:communities/update-last-opened-at community-id]]
              [:dispatch
-              [:communities/fetch-community
-               {:community-id           community-id
-                :update-last-opened-at? true}]]
+              [:communities/fetch-community {:community-id community-id}]]
              [:dispatch [:navigate-to :community-overview community-id]]
              (when (get-in db [:communities community-id :joined])
                [:dispatch
@@ -358,10 +353,7 @@
   (let [community-id (or community-id (get-in db [:chats chat-id :community-id]))]
     (merge
      {:fx [(when community-id
-             [:dispatch
-              [:communities/fetch-community
-               {:community-id           community-id
-                :update-last-opened-at? true}]])
+             [:dispatch [:communities/fetch-community {:community-id community-id}]])
            (if pop-to-root?
              [:dispatch [:chat/pop-to-root-and-navigate-to-chat chat-id]]
              [:dispatch

--- a/src/status_im/contexts/communities/events_test.cljs
+++ b/src/status_im/contexts/communities/events_test.cljs
@@ -49,7 +49,7 @@
         (testing "dispatch fxs"
           (is (match?
                {:fx [[:dispatch [:communities/handle-community {:id community-id}]]
-                     [:dispatch [:communities/update-last-opened-at community-id]]
+                     [:dispatch [:chat.ui/spectate-community community-id]]
                      [:dispatch
                       [:chat.ui/cache-link-preview-data "community-link+community-id"
                        {:id community-id}]]]}
@@ -60,7 +60,7 @@
         (testing "dispatch fxs, do not spectate community"
           (is (match?
                {:fx [[:dispatch [:communities/handle-community {:id community-id}]]
-                     [:dispatch [:communities/update-last-opened-at community-id]]
+                     [:dispatch [:chat.ui/spectate-community community-id]]
                      [:dispatch
                       [:chat.ui/cache-link-preview-data "community-link+community-id"
                        {:id community-id}]]]}
@@ -71,7 +71,7 @@
         (testing "dispatch fxs, do not spectate community"
           (is (match?
                {:fx [[:dispatch [:communities/handle-community {:id community-id}]]
-                     [:dispatch [:communities/update-last-opened-at community-id]]
+                     [:dispatch [:chat.ui/spectate-community community-id]]
                      [:dispatch
                       [:chat.ui/cache-link-preview-data "community-link+community-id"
                        {:id community-id}]]]}

